addons/controller_icons/ControllerIcons.gd

```diff
@@ -574,10 +574,20 @@ func _convert_key_to_path(scancode: int):
 			return "key/bracket_left"
 		KEY_BACKSLASH:
 			return "key/slash"
+		KEY_SLASH:
+			return "key/forwardslash"
```

Comment:
For consistency with other asset names, please add an underscore to separate these words (rename the icon as well):
```suggestion
			return "key/forward_slash"
```